packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -79,7 +79,7 @@ export class Span implements types.Span, ReadableSpan {
 
   addEvent(name: string, attributes?: types.Attributes): this {
     if (this._isSpanEnded()) return this;
-    this.events.push({ name, attributes });
+    this.events.push({ name, attributes, time: Date.now() });
```

Comment:
Aren't we using `performance.now()` for timestamps since it's monotonic and higher accuracy?